amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -606,6 +637,50 @@ describe('SearchPage', () => {
     });
   });
 
+  describe('renderContent', () => {
+    let content;
+    let props;
+    let wrapper;
+    describe('popular tables', () => {
+      beforeAll(() => {
+        const setupResult = setup({ searchTerm: '' });
+        props = setupResult.props;
+        wrapper = setupResult.wrapper;
+        content = shallow(wrapper.instance().renderContent());
+      });
+      
+      it('renders popular tables', () => {
+        expect(wrapper.instance().renderContent()).toEqual(wrapper.instance().renderPopularTables());
```

Comment:
nit: Another approach would be to spy on `renderPopularTables` and `renderSearchResults` and check that they were called. Imagine if we had thousands of tests with several hundred of them doing similar checks -- spying on methods whenever we can is expected be faster than if those hundreds of tests were rendering content and checking the output each time. Fine to merge as is though if you like.